src/gen_dbus.erl

```diff
@@ -261,21 +255,20 @@ do_method_call(Module, Member, Message = #dbus_message{header = Header}, Conn, S
 	end,
 
     Signature =
-        case lists:keysearch(signature, 1, Module:Member(dbus_info)) of
+        case lists:keysearch(signature, 1, erlang:apply(Module, Member, [dbus_info])) of
             {value, {signature, _Args, Returns}} ->
                 Returns;
             false ->
                 []
         end,
 
     Args =
-        case Message#dbus_message.body of
-            undefined -> [];
+        case Message#dbus_message.body of undefined -> [];
```

Comment:
just for curisioty: why this syntax change ?

Reply:
Ja, good question of which I have no answer, I shall revert that change.

Reply:
Cool so I've correct that change.